sim/SPI_v3/test/Loopback_test.py

```diff
@@ -0,0 +1,70 @@
+'''
+==========================================================================
+Loopback_test.py
+==========================================================================
+Unit test for Loopback.
+'''
+
+from pymtl3 import *
+from pymtl3.stdlib.test_utils import config_model_with_cmdline_opts
+
+from ..components.Loopback import Loopback
+
+# Helper function
+def t( dut, recv_val, recv_rdy, send_val, send_rdy, recv_msg, send_msg):
+
+  # Write input value to input port
+  dut.recv.msg        @= recv_msg
+  dut.recv.val        @= recv_val
+  dut.send.rdy        @= send_rdy
+
+  dut.sim_eval_combinational()
+
+  if recv_rdy != '?':
+    assert dut.recv.rdy == recv_rdy
+  
+  if send_msg != '?':
+    assert dut.send.msg == send_msg
+
+  if send_val != '?':
+    assert dut.send.val == send_val
+
+  # Tick simulator one cycle
+  dut.sim_tick()
+
+# # Test vectors
+
+
+def test_basic(cmdline_opts):
+  '''
+  This test is for Loopback(32)
+  '''
```

Comment:
Let's put the comments before `def test_basic`

Reply:
Moved comments before function definition